sensirion_arch_config.h

```diff
@@ -38,6 +38,16 @@
  */
 #include <stdint.h>
 
+/**
+* If your platform does not provide the library stdlib.h you have to
+* define NULL yourself (see below).
+ */
+#include <stdlib.h>
+
+#ifndef NULL
+#define NULL ((void *)0)
+#endif
```

Comment:
Should we comment it out per default and mention that the user should uncomment if `stdlib.h` is not available?

Reply:
I'd be ok with that

Reply:
Sure

Reply:
Done.

Reply:
Done.